wherescape/connectors/gitlab/gitlab_wrapper.py

```diff
@@ -39,7 +39,7 @@ def make_request(self, url, method, payload={}):
         response = requests.request(method, url, data=payload, headers=headers)
         return response
 
-    def format_url(self, resource_api, page_variables, simple, order_by, since=None):
+    def format_url(self, resource_api, page_variables, simple, order_by, sort, since=None):
```

Comment:
You're adding a parameter without making it optional. Are you sure that this function is called nowhere else?